examples/catalog/bin/sample_page.dart

```diff
@@ -136,7 +137,8 @@ class SampleInfo {
     commentValues['path'] = 'examples/catalog/${sourceFile.path}';
     commentValues['source'] = sourceCode.trim();
     commentValues['link'] = link;
-    commentValues['android screenshot'] = 'https://storage.googleapis.com/flutter-catalog/$commit/${sourceName}_small.png';
+    commentValues['android screenshot'] =
+        'https://storage.googleapis.com/flutter-catalog/$commit/${sourceName}_small.png';
```

Comment:
this is an example that i think is a blocker. it makes one line suddenly different from the other lines. They should either all have the line break, or none should, IMHO. In particular, this specific wrapping doesn't improve readability sufficiently to offset the pain caused by the inconsistency with the other lines.

Reply:
I agree that it's difficult to do this automatically. This is why we don't use a formatter.

Going off the side of the screen during code review is a one-time cost, easily resolved (and most monitors are far wider than 120 characters so not much of a cost at that). Having one line be different (or indeed all lines have line breaks as you show in your last example) is a cost that affects every single reader every single time. That's what we're trying to optimize for.

Having the one string literal go off the side of the screen - especially if it just contains a harmless URL with some trivial substitution - is not that much of a problem because readers can trust that the code author and code reviewer made sure that there was nothing controversial there so they don't need to go examine it to understand what the flow of the program is.

Reply:
I added a rule to prevent split.